vmtool/aws.py

```diff
@@ -4439,11 +4440,11 @@ def cmd_modify_disks(self, vm_id):
                     if newtype in ('io1', 'io2') and not modify_args.get('Iops'):
                         if not vol_conf.get('iops'):
                             eprintf("WARNING: cannot modify to %s without specifying IOPS: vol_name=%s", newtype, vol_name)
-                            continue
+                            skip = True
```

Comment:
Perhaps keep the `continue` here as next line tries to access `vol_conf['iops']`

Reply:
Done.